web/index.js

```diff
@@ -117,6 +133,7 @@ const handleDataActionDispatcher = {
   },
   'onCallBotData': (data) => {
     currentVariables = JSON.parse(data.detail.value);
+    parseIncidentCommanderInCurrentMessage();
```

Comment:
What will happen if the parsing here fails? 
Should it continue?

Reply:
yep. This function just converts the string to an object. If it fails, the only problem we will have is that the handlebars template won't show the IC on the page message.